webapp/public/workers/upload-worker.js

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable */
+self.onmessage = (e) => {
+  const { data } = e;
+  if (data && data.length > 0) {
+    // Upload local data to cloud
+    const headers = new Headers({
+      "Content-Type": "application/json",
+    });
+
+    // TODO: Figure out a way to remove the post dupe code
+    fetch(
+      "https://forestgeodataapi.azurewebsites.net/api/Census?code=xqRLaAgGAAQkMuMUwFu//JKC7BCEraubIlmfWOZsSBs4IsdbPDMF8w==",
```

Comment:
This looks like a secret... We can't check this into our repo. We'll need to come up with some authentication system in the future, but for now can we just change this to `"<----- insert connection string here ----->"`

Reply:
Leaving in secrets for agility, will clean up later.

Reply:
as discussed in offline conversation, due to the app doesn't have auth enabled, we are not able to find a secure way to calling function app without compromised the secret. (Everything in JavaScript or sent over by JavaScript is insecure). The plan is to roll the secret after hackathon ASAP and enable authentication soon. It requires conversation with Smithsonian.